backend/fleet_management/templates/new_drive_email.html

```diff
@@ -0,0 +1,28 @@
+Hello {{ passenger.first_name }}!
+Thanks for driving with me on {{ drive.date }} from {{ drive.start_location }} to {{ drive.end_location }}.
+Please confirm your participation and select your project by clicking on the link below.
+
+LINK
+
+Sincerely, {{ driver.first_name }} {{ driver.last_name }}
+
+---------------------------------------------------------
+
+привіт {{ passenger.first_name }}!
+Дякую за кермо зі мною у  вівтор, з {{ drive.start_location }} до {{ drive.end_location }}.
+Будь ласка, підтвердіть свою участь і вибрати проект, натиснувши на посилання нижче.
+
+LINK
+
+щиро, {{ driver.first_name }} {{ driver.last_name }}
+
+---------------------------------------------------------
+
+Witaj {{ passenger.first_name }}!
+Dziękuję za przejazd ze mną {{ drive.date }} z {{ drive.start_location }} do {{ drive.end_location }}.
+Potwierdź swój udzial i wybierz project klikając na link poniżej.
```

Comment:
Please add polish characters

Reply:
I can't :D I have "option + l" mapped in my IDE. Can you turn a blind eye here :D ?

Reply:
That sounds good - after all, all translations should be reviewed by a non dev person.
Iam wondering though if we should put all of the translations into one common file placed in the root of the project, rather then having one for backend and one for frontned. Should we create a task for that? What do you think?